### DISCUSSION TITLE
Native Homekit Support

### DISCUSSION THREAD

**Discussion by carzine** [gtxaspec/wz_mini_hacks#412] [General]
Any plans on adding native Homekit Support Directly from the Camera? This way there is no need to run another instance of Homebridge to load all the RTSP steams and then add it to Homekit. It seems to take up a decent amount of CPU once you get over 10 cameras.

**Comment by jmbulan** [comment]
i recently isntall Scrypted on my RPI4 running along side HA. the feeds load almost instally in HSV

**Reply by carzine** [reply]
How many streams do you have loaded? I don't have an issue with loading the RTSP streams then connecting it to Homekit via scripted. I think the only issue is that for every camera you add you Scrypted, it starts to eat up a decent amount of CPU even on idle. Which is why if there was native Homekit support on the Cameras, it wouldn't need to eat up extra CPU cycles on an RPI or a server.
